setup.py

```diff
@@ -43,6 +45,11 @@ def git_version():
             # Number of commits since the beginning of the current branch
             count = check_output("git rev-list --count HEAD".split()).decode('utf-8').strip()
 
+      # this will be a .dev version; increment the patchlevel
+      xyz = (tag.split('.') + [ '0', '0' ])[:3]
```

Comment:
This took me a long time to understand. I had to play with in in an interpreter to get convinced.

Using [the `packaging` module](https://packaging.pypa.io/en/latest/version/) seems a little more right, but the package produces immutable `Version` objects so you'd just do something similar again.

I think a little function with an `if-else` instead of the `+ ['0', '0']` trick would be preferable.

Reply:
Thanks! I reworked that part of the code to make it more readable.

I'm trying to avoid using external packages (`packaging` or `setuptools_scm`) -- they'd have to be manually installed by a developer before they can run `conda build` (otherwise mysterious errors arise). This may not be much of a big deal, but is one less thing to worry about if it's not needed.

Reply:
You're right of course about `packaging`. I had thought it was in the Python standard library, but it isn't.